.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+const config = {
```

Comment:
Is this the default or did you derive this from somewhere?

Reply:
Got the default config from prettier doc and changed some of the config so that prettier and eslint are not conflict with each other.